README.md

```diff
@@ -11,16 +11,23 @@ Chrome Platform Status
 
 ### Installation
 1. Before you begin, make sure that you have a java JRE (version 8 or greater) installed. JRE is required to use the DataStore Emulator.
-1. Install global CLIs
+1. Install global CLIs in the home directory
     1. [Google App Engine SDK for Python](https://cloud.google.com/appengine/docs/standard/python3/setting-up-environment). Make sure to select Python 3.
     1. pip, node, npm.
```

Comment:
Installing pip is problematic... See below.  
Add nvm, since it is used for using node 10.